content/integrations/uwsgi.md

```diff
@@ -0,0 +1,76 @@
+---
+categories:
+  - log collection
+  - web 
+doc_link: https://docs.datadoghq.com/integrations/uwsgi/
+description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+short_description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+ddtype: check
+git_integration_title: uwsgi
+has_logo: true
+integration_title: uwsgi
+is_public: true
+kind: integration
+name: uwsgi
+public_title: Integration Datadog-uWSGI
+short_description: 
+supported_os:
+- linux
+- mac_os
+- windows
+---
+
+## Overview
+
+Collect uWSGI logs in order tracks requests per second, bytes served, request status (2xx, 3xx, 4xx, 5xx), service uptime, slowness, and more.
+
+## Setup
+
+### Installation
+
+[Install the agent][1] on the instance that runs the uWSGI server.
+
+### Configuration
+
+By default uWSGI server logs to stdout. Run the following command to start logging to a file or follow [uWSGI instructions to log to a file][2]:
+
+```
+uwsgi --socket :3031 --logger file:logfile=/var/log/uwsgi/uwsgi.log,maxsize=2000000
+```
+
+Create the `uwsgi.d/conf.yaml` file in the at the root of your Agent's configuration directory.
```

Comment:
Create the `uwsgi.d/conf.yaml` file in the root of your Agent's configuration directory.